testcases/kernel/syscalls/timer_settime/timer_settime03.c

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
The default license for LTP is GPL-2.0-or-later, please use it unless you have a strong reason not to.